src/components/Calculator/Calculator.js

```diff
@@ -2,7 +2,9 @@ import React from "react";
 import TimeParser from "../../helper/TimeParser"
 
 const Calculator = ({ exerciseList }) => {
+  // we *could* use "constant-case" here -> EXERCISE_TIME and move this variable out of the function
```

Comment:
This is just a convention, but I find it makes code a lot clearer once you're used to it!